src/meltano/cli/invoke.py

```diff
@@ -90,29 +100,43 @@ def invoke(
     _, Session = project_engine(project)  # noqa: N806
     session = Session()
     plugins_service = ProjectPluginsService(project)
-    plugin = plugins_service.find_plugin(
-        plugin_name, plugin_type=plugin_type, invokable=True
-    )
+
+    try:
+        plugin = plugins_service.find_plugin(
+            plugin_name, plugin_type=plugin_type, invokable=True
+        )
+    except PluginNotFoundError as err:
+        with tracker.with_contexts(cmd_ctx):
+            tracker.track_command_event(cli_tracking.ABORTED)
```

Comment:
@pnadolny13 similar to `meltano run` if you pass a bad plugin to invoke I'll fire a `aborted` event, reserving `failed` for actual failures stemming from errors.